packages/apps-config/src/api/spec/dataavail.ts

```diff
@@ -85,12 +85,12 @@ const definitions: OverrideBundleDefinition = {
         KateCommitment: {
           rows: 'Compact<u16>',
           cols: 'Compact<u16>',
-          dataRoot: 'H256',
-          commitment: 'Vec<u8>'
+          commitment: 'Vec<u8>',
+          dataRoot: 'H256'
         },
         V1HeaderExtension: {
-          commitment: 'KateCommitment',
-          appLookup: 'DataLookup'
+          appLookup: 'DataLookup',
+          commitment: 'KateCommitment'
         },
         VTHeaderExtension: {
```

Comment:
Should it be changed too ? Just for the sake of completion ?

Reply:
Both Kate Commitment and Header Extension needed change? @kroos47

Reply:
Its being done in another place where types are overriden